### DISCUSSION TITLE
Settings Search

### DISCUSSION THREAD

**Discussion by maribox** [obsproject/obs-studio#11628] [General]
Hey!
I was looking for the settings to remux recordings and wondered why the Settings don't have search. Was this discussed in the past? Should I make an Issue about this?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!

**Reply by maribox** [reply]
It's not a support request. I thought about proposing a search feature in the OBS settings. 
I just wanted to name an example why I thought this could be relevant.
Should I open an issue?

**Reply by mihawk90** [reply]
Feature requests are handled on the [Ideas page](https://ideas.obsproject.com/), not on GitHub.

**Reply by maribox** [reply]
Thanks, sorry about that!
